include/great_expectations/expectations/taxi/demo.json

```diff
@@ -1,24 +1,36 @@
 {
-  "data_asset_type": "Dataset",
+  "data_asset_type": null,
   "expectation_suite_name": "taxi.demo",
   "expectations": [
     {
+      "expectation_context": {
+        "description": null
+      },
       "expectation_type": "expect_table_row_count_to_be_between",
+      "ge_cloud_id": null,
```

Comment:
I merged in a PR earlier today that should get rid of `null` ge_cloud_id's before they are dumped. I don't think this matters too much but just FYI